test/__snapshots__/validation.test.ts.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[` validation 1`] = `true`;
```

Comment:
What's this 😅

Reply:
This snapshot file is generated when you run the test once, make sure we always get same results.